lib/msal-browser/docs/configuration.md

```diff
@@ -70,6 +70,7 @@ const msalInstance = new PublicClientApplication(msalConfig);
 | `postLogoutRedirectUri` | URI that is redirected to after a logout() call is made. | String in URI format | Login request page (`window.location.href` of page which made auth request) |
 | `navigateToLoginRequestUrl` | If `true`, will navigate back to the original request location before processing the authorization code response. If the `redirectUri` is the same as the original request location, this flag should be set to false. | boolean | `true` |
 | `clientCapabilities` | Array of capabilities to be added to all network requests as part of the `xms_cc` claims request | Array of strings | [] |
+| `protocolMode` | Enum representing the protocol mode to use. If `"AAD"`, will function on the AAD v2 endpoints; if `"OIDC"`, will function on OIDC-compliant endpoints. | string | `"AAD"` |
```

Comment:
Let's validate that you can enter an AAD authority in OIDC mode and have it work - we're OIDC compliant, this is a good check.

Reply:
This is a good point, because currently it actually won't work, which kind of goes against the idea of this. Basically, if `protocolMode == "AAD"`, we append `v2.0/.well-known/openid-configuration/` to the url, and if `protocolMode == "OIDC"`, we append `./well-known/openid-configuration/`. I'll see what we can do here.

Reply:
Can we document this in that case?

Reply:
Pushed some changes to reflect that the AAD protocolMode is also still OIDC-compliant.